src/utils/apiHelpers.ts

```diff
@@ -37,13 +37,22 @@ async function fetchHelper(
     disableCache: !context.shouldUseCache,
   });
 
-  const authHeaders = context.token
+  const accessTokenAuth = context.token
     ? { Authorization: `Bearer ${context.token.access_token}` }
-    : {};
+    : null;
+
+  const feideAuthorization = context.feideAuthorization
+    ? { feideAuthorization: context.feideAuthorization }
+    : null;
+
   const cacheHeaders = !context.shouldUseCache
     ? { 'Cache-Control': 'no-cache' }
     : {};
-  const headers = { ...cacheHeaders, ...authHeaders };
+
+  const headers = {
+    ...(feideAuthorization || accessTokenAuth),
```

Comment:
Kan det være tilfeller der du skal sende både Authoriation og feideauthoruzation?

Reply:
det veit jeg ikke.

Reply:
Kanskje lurt å ta høgde for det då.

Reply:
Kan du legge inn dette? Så er vi sikre på at vi ikkje må endre i etterkant dersom det skulle vise seg at vi treng begge deler.